lib/cli/src/generators/ANGULAR/angular-helpers.ts

```diff
@@ -50,9 +50,23 @@ export function editStorybookTsConfig(tsconfigPath: string) {
   writeFileAsJson(tsconfigPath, tsConfigJson);
 }
 
-export function isDefaultProjectSet() {
-  const angularJson = readFileAsJson('angular.json', true);
-  return angularJson && !!angularJson.defaultProject;
+export function getDefaultProjectName(angularJson: any): string {
+  const { defaultProject, projects } = angularJson;
+
+  if (projects.storybook) {
```

Comment:
Even if you create an empty workspace `ng new my-project-name --create-application false`, `projects` won't be undefined. The `angular.json` will have this [format](https://github.com/storybookjs/storybook/blob/next/app/angular/src/server/__mocks-ng-workspace__/empty-projects-entry/angular.json).

However, I added some checks just in case